src/bindings/js/node/src/tensor.cpp

```diff
@@ -66,6 +66,12 @@ Napi::Object TensorWrap::wrap(Napi::Env env, ov::Tensor tensor) {
 }
 
 Napi::Value TensorWrap::get_data(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "getData() does not accept any arguments.");
+        return env.Undefined()
```

Comment:
```suggestion
        return env.Undefined();
```
there is a semicolon missing ;) I recommend checking `git status` carefully, building the project and running tests before pushing changes

Reply:
Done @almilosz